src/QUESTIONS.md

```diff
@@ -0,0 +1,401 @@
+## General
+
+### What CHERI is?
+
+See
+[An Introduction to CHERI](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-941.pdf).
+
+
+### What CHERI-extended hardware is available to use?
+
+[Arm Morello](https://www.arm.com/architecture/cpu/morello)
+is currently the only System-on-Chip with a CHERI-extended CPU.
+
+There are several
+[CHERI-RISC-V prototypes on FPGA](https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/cheri-risc-v.html),
+including a superscalar, out-of-order Toooba-based core.
+
+Additionally, CHERI-RISC-V and Morello can be emulated using QEMU-CHERI or
+Morello FVP.
+
+
+### How can I get an Arm Morello board?
+
+Go to the
+[Digital Security by Design](https://www.dsbd.tech/get-involved/morello-board-request/)
+website and follow the instructions there.
+
+
+### How can I emulate CHERI?
+
+Go to the
+[cheribuild](https://github.com/CTSRD-CHERI/cheribuild)
+repository and follow the instructions there.
```

Comment:
The instructions are rather long, I wonder if we should provide some succinct guidance here.